src/theme/components.ts

```diff
@@ -37,6 +37,10 @@ export const components: ISectionTheme = {
       border: '#/colors/transparent',
     },
 
+    scrollbar: {
```

Comment:
Could we add bg to https://github.com/stoplightio/ui-kit/blob/master/src/theme/components.ts#L23#L29?
I know it's consumed in a different component, but still, we could try to add it there? WDYT?
If that's not the case, I would consider renaming it to scrollContainer or something similar just to make it more clear.

Reply:
I think that we might want to remove the scrollbars section in the top level of the theme. Allowing a user to change width and radius of the thumb is too much customizability. Really just allow users to change color

Reply:
Agreed.

Reply:
Agreed - will leave it to you @casserni.

Reply:
Actually this gets back to the whole separating out "user customizable" portion of theme thing. We do want to maintain consistency with the look and feel of these scrollbars and https://github.com/stoplightio/ui-kit/blob/master/src/ListScroller.tsx#L69. Is there a better way than via theme variables that is not hardcoding duplicate values in both places?